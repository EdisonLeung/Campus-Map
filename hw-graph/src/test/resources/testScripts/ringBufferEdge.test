# Test adding 2 edges that point to 2 different nodes

# Create graph
CreateGraph ringBuffer

# Add 2 nodes
AddNode ringBuffer a
AddNode ringBuffer b

# Add the 2 edges pointing to the different nodes
AddEdge ringBuffer a b AB
AddEdge ringBuffer b a BA

# Print the nodes and the children
ListNodes ringBuffer
ListChildren ringBuffer a
ListChildren ringBuffer b