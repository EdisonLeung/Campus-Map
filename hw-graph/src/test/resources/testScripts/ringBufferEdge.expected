# Test adding 2 edges that point to 2 different nodes

# Create graph
created graph ringBuffer

# Add 2 nodes
added node a to ringBuffer
added node b to ringBuffer

# Add the 2 edges pointing to the different nodes
added edge AB from a to b in ringBuffer
added edge BA from b to a in ringBuffer

# Print the nodes and the children
ringBuffer contains: a b
the children of a in ringBuffer are: b(AB)
the children of b in ringBuffer are: a(BA)